AsyncDisplayKit/Private/ASLayoutTransition.mm

```diff
@@ -72,10 +72,16 @@ - (instancetype)initWithNode:(ASDisplayNode *)node
   return self;
 }
 
+- (instancetype)init
+{
+  ASDisplayNodeAssert(NO, @"Use the designated initializer");
+  return [self init];
```

Comment:
In this case you cannot call `[super init]` as `init` is in this case a `convenience initalizer` and cannot invoke an initializer on super